src/test/java/net/snowflake/client/jdbc/RestRequestWiremockIT.java

```diff
@@ -0,0 +1,75 @@
+package net.snowflake.client.jdbc;
+
+import java.util.concurrent.atomic.AtomicBoolean;
+import net.snowflake.client.category.TestCategoryOthers;
+import net.snowflake.client.core.ExecTimeTelemetryData;
```

Comment:
Do we know why there's some failing test regarding this line?
```
 Compilation failure: Compilation failure: 
Error:  /mnt/host/src/test/java/net/snowflake/client/jdbc/RestRequestWiremockIT.java:[5,33] cannot find symbol
```

Reply:
The name of the test influences how it's build and run. Changing name to one that ends in LatestIT fixed the issue (this test was 'inspired by' ProxyLatestIT)